frontend/src/components/Authentication/SignUp/SignUp.js

```diff
@@ -24,7 +24,7 @@ function SignUp(props) {
 
     let [firstName, setFirstName] = useState('');
     let [lastName, setLastName] = useState('');
-    let[country, setCountry] = useState('');
+    let[dateOfBirth, setDateOfBirth] = useState('');
```

Comment:
The date of birth value should be sent in the POST request body to be written in the database. Check the back-end route to name the parameter properly